mne/filter.py

```diff
@@ -11,7 +16,7 @@
 from .parallel import parallel_func, check_n_jobs
 from .cuda import (setup_cuda_fft_multiply_repeated, fft_multiply_repeated,
                    setup_cuda_fft_resample, fft_resample, _smart_pad)
-from .utils import logger, verbose, sum_squared, check_version
+from .utils import logger, verbose, sum_squared, check_version, _get_mkl_fft
```

Comment:
So in mne.utils, I'd have something like

```
def fft(*args, **kwargs):
    try:
        from mklfft.fftpack import fft
    except ImportError:
        from scipy.fftpack import fft
    return fft(*args, **kwargs)
```

... or if not, can you point me to some code that does what you mean?

I tried modeling this after how ica and xdawn handle fast_dot.